projects/packages/blocks/src/class-blocks.php

```diff
@@ -111,6 +111,11 @@ function () use ( $feature_name, $method_name ) {
 			}
 		}
 
+		// Keep track of the JS loading strategy for any block that specifies it.
+		if ( isset( $args['js_loading_strategy'] ) && class_exists( 'Jetpack_Gutenberg' ) ) {
+			Jetpack_Gutenberg::set_block_js_loading_strategy( $feature_name, $args['js_loading_strategy'] );
```

Comment:
Instead of checking for `class_exists( 'Jetpack_Gutenberg' )`, would it make sense to move that check inside the `if ( ! self::is_standalone_block() ) {` just above, which effectively does the same thing?